tests/Tests/ORM/Tools/Console/Command/Debug/DebugEntityListenersDoctrineCommandTest.php

```diff
@@ -0,0 +1,146 @@
+  Order   Listener                                                                             
+ ------- ------------------------------------------------------------------------------------- 
+  #1      Doctrine\Tests\ORM\Tools\Console\Command\Debug\Fixtures\BazListener::postPersists()  
+ ------- ------------------------------------------------------------------------------------- 
+
+"preUpdate" event
+-----------------
+
+ ------- ---------------------------------------------------------------------------------- 
+  Order   Listener                                                                          
+ ------- ---------------------------------------------------------------------------------- 
+  #1      Doctrine\Tests\ORM\Tools\Console\Command\Debug\Fixtures\FooListener::preUpdate()  
+  #2      Doctrine\Tests\ORM\Tools\Console\Command\Debug\Fixtures\BarListener::__invoke()   
+ ------- ---------------------------------------------------------------------------------- 
+
+
+TXT
+            , $commandTester->getDisplay(true));
+    }
+
+    public function testExecuteWithEvent(): void
+    {
+        $commandTester = new CommandTester($this->command);
+        $commandTester->execute(
+            ['command' => $this->command->getName(), 'entity' => self::class, 'event' => 'postPersists'],
+        );
+
+        self::assertSame(<<<'TXT'
+
+Entity listeners for Doctrine\Tests\ORM\Tools\Console\Command\Debug\DebugEntityListenersDoctrineCommandTest
+===========================================================================================================
+
+"postPersists" event
+--------------------
+
+ ------- ------------------------------------------------------------------------------------- 
+  Order   Listener                                                                             
+ ------- ------------------------------------------------------------------------------------- 
+  #1      Doctrine\Tests\ORM\Tools\Console\Command\Debug\Fixtures\BazListener::postPersists()  
+ ------- ------------------------------------------------------------------------------------- 
+
+
+TXT
+            , $commandTester->getDisplay(true));
+    }
+
+    public function testExecuteWithMissingEvent(): void
+    {
+        $commandTester = new CommandTester($this->command);
+        $commandTester->execute(
+            ['command' => $this->command->getName(), 'entity' => self::class, 'event' => 'preRemove'],
+        );
+
+        self::assertSame(<<<'TXT'
+
+Entity listeners for Doctrine\Tests\ORM\Tools\Console\Command\Debug\DebugEntityListenersDoctrineCommandTest
+===========================================================================================================
+
+"preRemove" event
+-----------------
+
+ No listeners are configured for this event.
+
+TXT
+            , $commandTester->getDisplay(true));
+    }
+
+    /** @return MockObject&ManagerRegistry */
+    private function getMockManagerRegistry(): MockObject
```

Comment:
```suggestion
    private function getMockManagerRegistry(): ManagerRegistry
```

Let's pick the more obvious type here and in the other tests.